modules/appeals_api/app/controllers/appeals_api/v1/decision_review/higher_level_reviews_controller.rb

```diff
@@ -1,22 +1,50 @@
 # frozen_string_literal: true
 
+require 'json_schema/json_api_missing_attribute'
+require 'appeals_api/form_schemas'
+require_dependency 'appeals_api/concerns/json_format_validation'
+
 class AppealsApi::V1::DecisionReview::HigherLevelReviewsController < AppealsApi::ApplicationController
+  include AppealsApi::JsonFormatValidation
+
   skip_before_action(:authenticate)
+  before_action :validate_json_format, if: -> { request.post? }
+  before_action :validate_json_schema, only: %i[create validate]
```

Comment:
these names threw me off for a sec. I was thinking `validate_json_schema` --he's making sure our json schema is valid with each create or validate call?
i don't know what would be better. i'm probably too long winded:
```
before_action :ensure_request_body_is_json
before_action :validate_request_json
```
(_not_ a blocking comment)